### PR TITLE
chore: update storybook to 5.3.3 & implement addon-docs - FE-1998

### PR DESCRIPTION
**Pull request by dawidzarzycki**
### Proposed behaviour
Add the new version of a storybook that allows use addon-docs to create better documentation

### Current behaviour
Older version of a storybook that doesn't support addon-info

### Checklist

- [x] Release notes
<del>- [ ] Unit tests
<del>- [ ] Cypress automation tests
- [x] Storybook added or updated
<del>- [ ] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Additional context
FE-1998

### Testing instructions
- Go to storybook
- pick any of component
- go to DOCS tab
- error screen shouldn't be visible
- screen with information about DOCS should be visible

### CODE REVIEW

File: change_log/next/FE-1998-storybook-update.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+Dependency Updates: Updates storybook to 5.2.8v. Adds storybook addon-docs.
```
Comment (ianforster-sage) [inline change_log/next/FE-1998-storybook-update.yml]:
```suggestion
Dependency Updates: Updates storybook to v5.2.8. Adds storybook addon-docs.
```

File: src/components/button-toggle/button-toggle.stories.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ function makeStory(name, themeSelector) {
     };
 
     return (
-      [
+      <>
```
Comment (ianforster-sage) [inline src/components/button-toggle/button-toggle.stories.js]: Using React Fragments like this Ie. declaring what look like empty elements causes the Storybook addon to output this message.
```
"Unknown" Component
No propTypes defined!
```
It's better to use a `<div>` here, as there's no problem for us to add the extra element to the DOM and it cleans up that Unknown Component output.

File: src/components/card/card.stories.js [hunk 3]
```diff
@@ -115,7 +115,11 @@ function makeStory(name, themeSelector) {
   const component = () => {
     const knobs = cardKnobs();
 
-    return ([getCard(knobs), getSmallCard(knobs)]);
+    return (
+      <>
```
Comment (ianforster-sage) [inline src/components/card/card.stories.js]: Same as my earlier comment about empty elements.

**Comment by edleeks87** [comment]
Do we still need a change_log now file now semantic releasing has been implemented?

**Comment by jamime** [comment]
@edleeks87 we do not, it is generated by `@semantic-release/changelog`.

File: src/components/button-toggle/button-toggle.stories.js (hunk 2 above)
Comment (ianforster-sage) [reply src/components/button-toggle/button-toggle.stories.js]: @damianOfKyan this issue has reappeared. Please could you resolve it.

File: src/components/card/card.stories.js (hunk 3 above)
Comment (ianforster-sage) [reply src/components/card/card.stories.js]: This has reappeared too.

File: src/components/button-toggle/button-toggle.stories.js (hunk 2 above)
Comment (damianOfKyan) [reply src/components/button-toggle/button-toggle.stories.js]: fixed in d1a866b

File: src/components/card/card.stories.js (hunk 3 above)
Comment (damianOfKyan) [reply src/components/card/card.stories.js]: fixed in d1a866b

**Comment by damianOfKyan** [comment]
> There is a message in the console `Please use '@storybook/addon-docs/preset' instead of '@storybook/addon-docs/react/preset'.`
fixed in 8115344

**Pull request merged**